frontend/src/Components/Dropdown/Dropdown.tsx

```diff
@@ -1,35 +1,72 @@
-import classnames from 'classnames';
+import { Icon } from '@iconify/react';
+import { default as classNames, default as classnames } from 'classnames';
+import { ChangeEvent } from 'react';
+import { ReactElement } from 'react-markdown/lib/react-markdown';
 import styles from './Dropdown.module.scss';
 
-type DropdownProps = {
+export type DropDownOption<T> = {
+  label: string;
+  value: T;
+};
+
+type DropdownProps<T> = {
   className?: string;
   wrapper?: string;
-  default_value?: string;
-  options?: string[][];
-  label?: string;
-  onChange?: (e?: React.ChangeEvent<HTMLSelectElement>) => void;
+  default_value?: DropDownOption<T>;
```

Comment:
Jeg foretrekker snake_case selv, men inntil vi har blitt enige om noe, skal vi prøve å holde vanlig standard?
```suggestion
  defaultValue?: DropDownOption<T>;
```

Reply:
Snake case burde være ulovlig i typescript, change my mind.
Siden camel case er standard burde vi bruke det. Mener jeg kun endret typen her og lot navnet være.

Reply:
Er bare vanesak/konvensjon, pascalCase er inferior til snake_case. Den ene er klart enklere å lese enn den andre ;)

Reply:
Glad i snake_case også, men kjenner smerte i hele kroppen når det blandes med camelCase og PascalCase.